core/templates/dev/head/pages/Base.ts

```diff
@@ -53,16 +53,13 @@ oppia.controller('Base', [
 
     $scope.isBackgroundMaskActive = BackgroundMaskService.isMaskActive;
 
+    CsrfTokenService.getToken();
```

Comment:
With a name like getToken(), you would expect this to actually return something.

So ... why isn't this doing anything with the return value? Perhaps you should break out some part of getToken() into initializeToken() or similar.

Reply:
Done